scripts/items/flask_of_distilled_water.lua

```diff
@@ -0,0 +1,26 @@
+-----------------------------------
+-- ID: 4509
+-- Item: Flask of distilled water
+-- Item Effect: While Sha Wujing's Lance +1 is equipped:
+--   5m 1HP/tick Regen effect. Lance does not need to remain equipped after use.
+-----------------------------------
+local itemObject = {}
+
+itemObject.onItemCheck = function(target)
+    return 0
+end
+
+itemObject.onItemUse = function(target)
+    local mainWeapon = target:getEquipID(xi.slot.MAIN)
+
+    if
+        mainWeapon == xi.item.SHA_WUJINGS_LANCE_P1 and
```

Comment:
this is a pattern we've specifically been against for as long as I can remember. as one point hundred of lines exactly like this were removed from the repo